.travis.yml

```diff
@@ -1,3 +1,7 @@
 language: node_js
 node_js:
+  - "5"
```

Comment:
Don't need to test against 5. I don't think we'll ever actual use 5 as 6 is LTS and 7 is the latest. We should probably leave it to "4" "6" and "node" which is the latest.